Makefile

```diff
@@ -1,14 +1,73 @@
+SHELL := /bin/bash
+
+NAME ?= terrahelp
+
 BUILDARGS ?= -mod=vendor
 
+BIN := $(CURDIR)/bin
+DIST := $(CURDIR)/dist
+OUPUT_FILES := $(BIN) $(DIST)
+
+PLATFORMS ?= darwin linux
+ARCH ?= amd64
+OS = $(word 1, $@)
+
 check:
 	go vet $(BUILDARGS) ./...
+.PHONY: check
 
 test:
 	go test $(BUILDARGS) -v ./...
+.PHONY: test
 
 build:
-	go build $(BUILDARGS) -o ${GOPATH}/bin/terrahelp
+	go build $(BUILDARGS) -o bin/$(NAME)
+.PHONY: build
+
+install:
+	go build $(BUILDARGS) -o ${GOPATH}/bin/$(NAME)
```

Comment:
[MINOR] Shall this make file have a check of the go version ~1.
13? as defined in the README.md, have see few examples and its painful. :(